test/protocol/MetaTransactionsHandlerTest.js

```diff
@@ -543,7 +543,7 @@ describe("IBosonMetaTransactionsHandler", function () {
           message.from = await assistant.getAddress();
           message.contractAddress = await accountHandler.getAddress();
           message.functionName =
-            "createSeller((uint256,address,address,address,address,bool,string),(uint256,uint8),(string,uint256))";
+            "createSeller((uint256,address,address,address,address,bool,string),(uint256,uint8),(string,uint256,bytes32))";
```

Comment:
just a note we have to make sure the metatx hash for this function is updated on migration script